src/fun_lti_provider/settings.py

```diff
@@ -0,0 +1,31 @@
+"""This module contains default settings for fun_lti_provider application"""
+
+from django.conf import settings
+
+# When a LTI launch request is successfully verified, the handler function defined
+# in the LTI_LAUNCH_SUCCESS_HANDLER setting is called with 2 parameters :
+# 1. The request (django.http.request.HttpRequest)
+# 2. The verifier LTI request (fun_lti_provider.lti.LTI)
```

Comment:
What about the following?
```suggestion
# 2. The LTI request validator (fun_lti_provider.lti.LTI)
```

Reply:
It's a typo, I meant **verified** LTI request